src/models/tools/tool-tile.ts

```diff
@@ -87,13 +87,13 @@ export const ToolTileModel = types
   .actions(self => ({
     afterCreate() {
       const metadata = findMetadata(self.content.type, self.id, self.title);
-      const content = self.content as any;
+      const content = self.content;
       if (metadata && content.doPostCreate) {
         content.doPostCreate(metadata);
       }
     },
     willRemoveFromDocument() {
-      const willRemoveFromDocument = (self.content as any).willRemoveFromDocument;
+      const willRemoveFromDocument = self.content.willRemoveFromDocument;
       return willRemoveFromDocument && willRemoveFromDocument();
```

Comment:
```suggestion
      return self.content.willRemoveFromDocument?.();
```

Reply:
I'll fix that in the following PR: https://github.com/concord-consortium/collaborative-learning/pull/1309